src/index.ts

```diff
@@ -10,140 +10,34 @@
  */
 
 import * as theia from '@wiptheia/plugin';
-import * as _ from 'lodash';
-import {initEditorsCommands} from './editor';
+import { COMMAND_NAME_PREFIX, CONSOLE_OUTPUT_COMMAND_SUFIX, CONSOLE_OUTPUT_PREFIX, PLUGIN_NAME } from './common/constants';
+import { initEditorsCommands } from './editor';
+import { initEnvCommands } from './env';
+import { initWindowStateCommands } from './window-state';
+import { initQuickPickCommands } from './quick-pick';
+import { initMessagesCommands } from './messages';
+
 const disposables: theia.Disposable[] = [];
 
 export function start() {
-    initEditorsCommands();
+    initEditorsCommands(disposables);
+    initEnvCommands(disposables);
+    initWindowStateCommands(disposables);
+    initQuickPickCommands(disposables);
+    initMessagesCommands(disposables);
+
+    // Hello World command
     const command: theia.Command = {
-        id: 'simple-frontend-plugin-command',
-        label: 'Command from simple plugin'
+        id: PLUGIN_NAME + 'simple-command',
+        label: COMMAND_NAME_PREFIX + 'Hello World command' + CONSOLE_OUTPUT_COMMAND_SUFIX
     };
     disposables.push(
         theia.commands.registerCommand(command, (...args: any[]) => {
-            console.log(`>>> Simple plugin command handler was called with arguments: `, args);
-
-            if (typeof (_ as any).all === 'undefined') {
-                console.log('>>> Lodash v4 is present');
-            }
+            console.log(CONSOLE_OUTPUT_PREFIX + `Hello World command handler was called with arguments: `, args);
```

Comment:
It will be better to create an own method for the logger and use it for test API logs. It helps us to change log's format in a simple way.
The best variant - extend our API for ILogger based on @theia/core/lib/common/logger and use it for logs for test plugin.

Reply:
Makes sense